### PR TITLE
Minor changes to MTV 2.4.2 release notes

### PR DESCRIPTION
**Pull request by RichardHoch**
2.4.2
 
 [no jira] Makes minor changes in the release notes for MTV 2.4.2. 
 
 Preview: https://file.emea.redhat.com/rhoch/rn_mods/html-single/#known-issues-24_release-notes, sections 1.3 and 1.4

### CODE REVIEW

**Comment by RichardHoch** [comment]
@ahadas  please review.

File: documentation/modules/rn-2.4.adoc [hunk 1]
```diff
@@ -97,11 +93,11 @@ The Migration Controller service does not delete snapshots that are created duri
...
 
-Some warm migrations from {rhv-short} might fail. When running a migration plan for warm migration of multiple VMs from {rhv-short}, the migrations of some VMs might fail during the cutover stage. In that case, restart the migration plan and set the cutover time for the VM migrations that failed in the first run. link:https://bugzilla.redhat.com/show_bug.cgi?id=2063531[(BZ#2063531)] link:https://issues.redhat.com/browse/MTV-456[(MTV-456)]
+Some warm migrations from {rhv-short} might fail. When running a migration plan for warm migration of multiple VMs from {rhv-short}, the migrations of some VMs might fail during the cutover stage. In that case, restart the migration plan and set the cutover time for the VM migrations that failed in the first run. link:https://issues.redhat.com/browse/MTV-456[(MTV-456)]
 
 Warm migration from {rhv-short} fails if a snapshot operation is performed on the source VM. If the user performs a snapshot operation on the source VM at the time when a migration snapshot is scheduled, the migration fails instead of waiting for the user’s snapshot operation to finish. link:https://bugzilla.redhat.com/show_bug.cgi?id=2057459[(BZ#2057459)]
```
Comment (ahadas) [inline documentation/modules/rn-2.4.adoc:102]: BZ#2057459 refers to a similar issue with snapshots that would be addressed by MTV-456 - therefore I closed it as MIGRATED to MTV-456 as well, we can also replace it

File: documentation/modules/rn-2.4.adoc [hunk 2]
```diff
@@ -163,3 +159,6 @@ The Universal Unique ID (UUID) number within the System Management BIOS (SMBIOS)
 
 Previously, the password that was specified for {rhv-short} manager appeared in error messages that were displayed in the web console and logs when failing to connect to {rhv-short}. In this release, error messages that are generated when failing to connect to {rhv-short} do not reveal the password for {rhv-short} manager.
 
+.QEMU guest agent is not installed on migrated VMs
+
+The QEMU guest agent is not installed on migrated VMs. Workaround: Install the QEMU guest agent with a post-migration hook. link:https://bugzilla.redhat.com/show_bug.cgi?id=2018062[(BZ#2018062)]
```
Comment (ahadas) [inline documentation/modules/rn-2.4.adoc:164]: this should rather now be something like:
```suggestion
The QEMU guest agent is installed on VMs during cold migration from vSphere. link:https://issues.redhat.com/browse/MTV-332[(MTV-332)]
```

Comment (RichardHoch) [reply documentation/modules/rn-2.4.adoc]: Fixed -- also changed "not" in header to "now".

Comment (RichardHoch) [reply documentation/modules/rn-2.4.adoc]: @anarnold97 2nd eyes on this? Would you go with "not installed," "now installed," or simply "installed" ?

**Review by ahadas — APPROVED** [review]

**Pull request merged**